src/roswire/ros2/state.py

```diff
@@ -1,8 +1,12 @@
 # -*- coding: utf-8 -*-
 __all__ = ('ROS2StateProbe',)
 
+import dockerblade
```

Comment:
Apologies for a very nitpicky comment. Imports should be grouped into three blocks: 1. standard library imports, 2. third-party imports, 3. relative imports. (See: https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
Oh I didn't know this. Thanks for telling me! I'll fix it for this and everything else